evm/src/cpu/kernel/asm/account_code.asm

```diff
@@ -1,8 +1,18 @@
 global sys_extcodehash:
     // stack: kexit_info, address
-    // TODO: Charge gas.
-    SWAP1
+    SWAP1 %u256_to_addr
     // stack: address, kexit_info
+    DUP1 %insert_accessed_addresses
+    // stack: address in access_list, address, kexit_info
```

Comment:
nit: to me `address in access_list` sounds like the access was warm, but it's the opposite right? I would say something like `cold_access` or `was_inserted`.

Reply:
Oh yes good point, this is misleading. Will change to `cold_access`.